solutions/swb-reference/src/SWBStack.ts

```diff
@@ -155,6 +155,29 @@ export class SWBStack extends Stack {
     this._createDDBTable(apiLambda, statusHandler, createAccountHandler);
     this._createRestApi(apiLambda);
 
+    const metadatanode = this.node.findChild('AWS679f53fac002430cb0da5b7982bd2287').node
```

Comment:
For follow up PR: could you make this a helper method so that you can pass in the child id, rules, and reasons to deduce duplication?